controlplane/pkg/plugins/utils.go

```diff
@@ -38,6 +39,32 @@ func StartPlugin(name, registry string, services map[plugins.PluginCapability]in
 		return err
 	}
 
+	go func() {
```

Comment:
I'd move the whole goroute func into separate func with name `monitorNSMD` or something like that

Reply:
- [ ] move the whole goroute func into separate func with name monitorNSMD